sdcm/utils/data_validator.py

```diff
@@ -430,43 +470,48 @@ def validate_range_expected_to_change(self, session, during_nemesis=False):
                 return
 
             before_update_rows = self.longevity_self_object.fetch_all_rows(
                 session=session, default_fetch_size=self.DEFAULT_FETCH_SIZE,
                 statement=f"SELECT {partition_keys} FROM {views_set[0]}",
                 verbose=not during_nemesis)
             if not before_update_rows:
-                LOGGER.error(f"Can't validate updated rows. "
-                             f"Fetch all rows from {views_set[0]} failed. See error above")
+                DataValidatorEvent(type='error', name='UpdatedRowsValidator', status=Severity.ERROR,
+                                   error=f"Can't validate updated rows. "
+                                         f"Fetch all rows from {views_set[0]} failed. See error above")
                 return
 
             after_update_rows = self.longevity_self_object.fetch_all_rows(
                 session=session, default_fetch_size=self.DEFAULT_FETCH_SIZE,
                 statement=f"SELECT {partition_keys} FROM {views_set[1]}",
                 verbose=not during_nemesis)
             if not after_update_rows:
-                LOGGER.error(f"Can't validate updated rows. "
-                             f"Fetch all rows from {views_set[1]} failed. See error above")
+                DataValidatorEvent(type='error', name='UpdatedRowsValidator', status=Severity.ERROR,
+                                   error=f"Can't validate updated rows. "
+                                         f"Fetch all rows from {views_set[1]} failed. See error above")
                 return
 
             expected_rows = self.longevity_self_object.fetch_all_rows(
                 session=session, default_fetch_size=self.DEFAULT_FETCH_SIZE,
                 statement=f"SELECT {partition_keys} FROM {views_set[2]}",
                 verbose=not during_nemesis)
             if not expected_rows:
-                LOGGER.error(f"Can't validate updated row. "
-                             f"Fetch all rows from {views_set[2]} failed. See error above")
+                DataValidatorEvent(type='error', name='UpdatedRowsValidator', status=Severity.ERROR,
+                                   error=f"Can't validate updated row. "
+                                         f"Fetch all rows from {views_set[2]} failed. See error above")
                 return
 
             # Issue https://github.com/scylladb/scylla/issues/6181
             # Not fail the test if unexpected additional rows where found in actual result table
             if len(before_update_rows) + len(after_update_rows) > len(expected_rows):
-                LOGGER.warning(f'View {views_set[0]}. '
-                               f'Actual dataset length {len(before_update_rows) + len(after_update_rows)} more '
-                               f'then expected dataset length: { len(expected_rows)}. Issue #6181')
+                DataValidatorEvent(type='info', name='UpdatedRowsValidator', status=Severity.NORMAL,
```

Comment:
better make it warning

Reply:
Changed to warning. But it's expected to see this warning every nemesis - may be 2 events: before and after nemesis.